tests/test_profile.py

```diff
@@ -86,11 +86,10 @@ def test_skill_deletion(self, base_url, selenium, vouched_user):
         skills_form.click_update()
 
         settings = home_page.header.click_settings_menu_item()
+        # TODO: Try to remove the sleep after https://bugzilla.mozilla.org/show_bug.cgi?id=1291320 is fixed.
```

Comment:
For this, might I recommend also filing an Issue in this GitHub repo noting to return to this?  Always good to have it visible (no pun intended) :-)